config/index.js

```diff
@@ -1,110 +0,0 @@
-"use strict"
-
-const path = require("path")
-
-module.exports = {
-  build: {
-    // Webpack mode
-    mode: "production",
-
-    // Template for index.html
-    index: path.resolve(__dirname, "../dist/index.html"),
-
-    // Paths
-    assetsRoot: path.resolve(__dirname, "../dist"),
-    assetsSubDirectory: "assets",
-    assetsPublicPath: "/",
-
-    // Source map
-    productionSourceMap: false,
-    // https://webpack.js.org/configuration/devtool/#production
-    devtool: "#source-map",
-
-    // Gzip off by default as many popular static hosts such as
-    // Surge or Netlify already gzip all static assets for you.
-    // Before setting to `true`, make sure to:
-    // npm install --save-dev compression-webpack-plugin
-    productionGzip: false,
-    productionGzipExtensions: ["js", "css"],
-
-    // Run the build command with an extra argument to
-    // View the bundle analyzer report after build finishes:
-    // `npm run build --report`
-    // Set to `true` or `false` to always turn it on or off
-    bundleAnalyzerReport: process.env.npm_config_report,
-  },
-  system: {
-    // Webpack mode
-    mode: "production",
-
-    // Template for index.html
-    index: path.resolve(__dirname, "../dist/index.html"),
-
-    // Paths
-    assetsRoot: path.resolve(__dirname, "../dist"),
-    assetsSubDirectory: "system",
-    assetsPublicPath: "/",
-
-    // The system.js library target, supports all
-    // Webpack libraryTargets (umd, commonjs2, etc...)
-    //
-    // In general you probably want this to be “umd”,
-    // but for SSR/Nuxt.js you might want to use “commonjs2”
-    // to avoid usage of window object.
-    libraryTarget: "umd",
```

Comment:
What are your thoughts, where would we be setting what options are used when a library build is done?

Reply:
When typing vue-cli-service --target lib, a umd library is created. Options are limited but all documented on the [vue cli website](https://cli.vuejs.org/)

[See documentation](https://cli.vuejs.org/guide/build-targets.html)

Reply:
I think that might be great if we don't have to manage that part at all. As long as things work.